test/e2e/bootstrapper_test.go

```diff
@@ -2,23 +2,23 @@ package e2e
 
 import (
 	"fmt"
+	"github.com/openshift/windows-machine-config-operator/pkg/bootstrapper"
+	"github.com/spf13/pflag"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
 	"io/ioutil"
 	"os"
 	"strings"
 	"testing"
 	"time"
-
-	"github.com/openshift/windows-machine-config-operator/pkg/bootstrapper"
-	"github.com/spf13/pflag"
-	"github.com/stretchr/testify/assert"
-	"golang.org/x/sys/windows/svc"
-	"golang.org/x/sys/windows/svc/mgr"
 )
 
 var ignitionFilePath string
 var kubeletPath string
 var installDir string
+var kubeletArgs map[string]string
```

Comment:
Why is this a global variable? It is not like you are populating it once and reusing it everywhere.

Reply:
Must have missed this - will make this local